low-level/sensors/impl/generic/node_hw.py

```diff
@@ -280,24 +280,22 @@ def initialize(self, conf_reader, msgQlist, products):
                 self._bmc_passwd = encryptor.decrypt(
                     decryption_key, _bmc_secret, self.SENSOR_NAME)
             except Exception as err:
-                logger.critical(f"BMC password decryption failed due to {err},"
+                raise Exception(f"BMC password decryption failed due to {err},"
                     "NodeHWSensor monitoring disabled.")
-                self.shutdown()
 
         # Set flag 'request_shutdown' to true if ipmitool/simulator is non-functional
         _, _, retcode = self._run_ipmitool_subcommand("sel info")
         if retcode != 0 and self.channel_err:
             if self._channel_interface == system:
-                log_msg = (
+                err_msg = (
                     "ipmitool commands not able to access BMC through"
                     " configured %s interface." % self._channel_interface)
             else:
-                log_msg = (
+                err_msg = (
                     "ipmitool commands not able to access BMC through"
                     " configured %s interface or even fallback option"
                     " 'KCS' interface." % (self._channel_interface))
-            logger.critical(log_msg)
-            self.request_shutdown = True
+            raise Exception(err_msg)
```

Comment:
`raise with e` as described in the previous file.

Reply:
As justified above.